scripts/check-imports

```diff
@@ -1,22 +1,27 @@
 #!/bin/sh
 
-# Generated type definitions should not include default imports.
-
 BASE_DIR="$(dirname "$0")/..";
 
-OUTPUT_FILE="/dist/react-stripe.d.ts"
+# Generated type definitions should not include default imports.
+function checkImport {
```

Comment:
(2)

```suggestion
checkImport() {
```
is more portable

Reply:
Yup it's blowing up CI apparently. My shell is a bit rusty :p